get prototypes for malloc/exit/strcpy

Patch by Mike Frysinger <vapier@gentoo.org>

--- a/dhry.h
+++ b/dhry.h
@@ -388,6 +388,8 @@
 /* General definitions: */
 
 #include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
                 /* for strcpy, strcmp */
 
 #define Null 0 
